README.md

```diff
@@ -8,3 +8,155 @@ development.
 
 This repo is under active development and is not ready for production (or even
 development) use, but stay tuned for updates!
+
+## Installing Spinnaker
+
+### Prerequisites
```

Comment:
Just finished all the prerequisites on my ☁️ 💻 ! Awarding myself the title "slowest kleat user" (for now).